js/filter-set.js

```diff
@@ -64,9 +63,9 @@ FilterSet.prototype.clear = function() {
   });
 };
 
-FilterSet.prototype.handleTagRemove = function(e) {
-  var $input = this.$body.find('#' + e.key);
-  var type = $input.attr('type');
+FilterSet.prototype.handleTagRemove = function(e, opts) {
+  var $input = this.$body.find('#' + opts.key);
+  var type = $input.attr('type') || 'text';
```

Comment:
Ah gotcha. I didn't realize that you could actually leave the `type` off an `input`.